Tests/UnitTests/Core/Propagator/BoundToCurvilinearConversionTests.cpp

```diff
@@ -197,6 +197,9 @@ void test_bound_to_curvilinear(const std::vector<TestData> &test_data_list,
       }
     }
 
+    FreeVector free_param_vec = Acts::detail::transformBoundToFreeParameters(
+        *surface, geoCtx, param_vec);
```

Comment:
```suggestion
    FreeVector freeParamVec = Acts::detail::transformBoundToFreeParameters(
        *surface, geoCtx, paramVec);
```

This file has odd variable naming. Do you know why?
Not sure, if we should keep the file consistent for now and change more of them in a later go?

Reply:
I think the code is just outdated